dist/main.js

```diff
@@ -0,0 +1,52 @@
+
+const model = new Model()
+const renderer = new Renderer()
+
+const searchForCity = async function () {
+    const cityName = $('input').val()
+    const results = await model.getCityWeather(cityName)
+    renderer.renderAllCities(results.cities)
+}
+
+const addCity = async function () {
+    const element = $(this).closest('.city')
+    const temp = element.find('.city-temp').html().split(" ")
```

Comment:
Why are you working hard on finding the specific information for each property? Just find it in the array of your model (the whole object is already there).